### ISSUE TITLE
Controls not showing up in HomeKit

### ISSUE THREAD

**Issue by Stuart-Richards** [ufodone/envisalink_new#106]
Firstly, thank you so much for rewriting the integration! Like others, I can't get the arming/disarming to work on the previous iteration with my Honeywell Vista-20p board. It's working great with this integration and everything is looking good in HA.

The issue I'm seeing is that nearly all the controls are not being exposed into HomeKit. The only control visible in HomeKit is the switch to enable or disable the panel Chime.

None of the sensors show, and neither does the control to actually enable or disable the alarm itself.

**Comment by ufodone** [comment]
Hi, I'm sorry but I don't use HomeKit so don't have much advice for you.  But I'm not aware of anything specific to the integration that would be preventing it.  I believe there are other users have been  using it with HomeKit successfully.  E.g. someone mentions it working for them in #51.

**Comment by gadgetguyj** [comment]
Hey @Stuart-Richards, I ran into this as well.

I'm assuming you're using the HomeKit Bridge integration with GUI config. I was able to use Siri to control and it showed up if I navigated directly to the bridge in HomeKit (Home Settings > Bridges > HASS Bridge), but didn't show up in the room.

I set up a second integration entry with YAML/manual configuration and included the alarm control panel there and everything functions as expected.

**Comment by Stuart-Richards** [comment]
@gadgetguyj, Thank you!

Yes, this pointed me in the right direction. I uninstalled and reinstalled the integration. Then a new Bridge showed up. I'm wondering if that new Bridge did show up originally and maybe I dismissed it not realizing it's significance.....  When I added the new Bridge to HomeKit, I see the Alarm control. Just the control, not any zones, but that's certainly good enough for me to use it! I only did the set up in the GUI, didn't create any YAML entries this time. But it's definitely working OK.

Curious why it would show up as a whole new Bridge, rather than be exposed as another Accessory within the existing Bridge, but I'll take it!

Thank you @gadgetguyj  and @ufodone!
